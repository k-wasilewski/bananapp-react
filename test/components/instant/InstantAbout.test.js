import React from 'react';
import renderer from 'react-test-renderer';
import store from "../../../src/redux/store";
import {BrowserRouter} from "react-router-dom";
import InstantAbout from "../../../src/components/instant/InstantAbout";
import {Provider} from "react-redux";

describe("InstantAbout rendering specification", () => {
    it('InstantAbout is rendered', () => {
        const component = renderer.create(
            <Provider store={store}>
                <BrowserRouter>
                    <InstantAbout />
                </BrowserRouter>
            </Provider>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});