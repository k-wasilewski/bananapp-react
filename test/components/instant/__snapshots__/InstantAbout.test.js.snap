// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstantAbout rendering specification InstantAbout is rendered 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <div
      className="col-13 text-left"
    >
      <h5>
        Kuba Wasilewski 
        <br />
        <a
          href="https://github.com/k-wasilewski"
        >
          @k-wasilewski 
        </a>
        <br />
        <a
          className="nav-link color-header"
          href="mailto:k.k.wasilewski@gmail.com"
        >
          k.k.wasilewski@gmail.com
        </a>
      </h5>
    </div>
    <a
      href="/"
      onClick={[Function]}
    >
      <button
        variant="outlined"
      >
        Back
      </button>
    </a>
  </header>
</div>
`;
